appium/tests/screenobjects/mail-folder.screen.ts

```diff
@@ -1,31 +1,63 @@
 import BaseScreen from './base.screen';
-import ElementHelper from "../helpers/ElementHelper";
 import TouchHelper from "../helpers/TouchHelper";
+import ElementHelper from "../helpers/ElementHelper";
 
 const SELECTORS = {
-  ENTER_YOUR_PASS_PHRASE_FIELD: '-ios class chain:**/XCUIElementTypeSecureTextField[`value == "Enter your pass phrase"`]',
-  OK_BUTTON: '~Ok',
-  CONFIRM_PASS_PHRASE_FIELD: '~textField',
+  TRASH_HEADER: '-ios class chain:**/XCUIElementTypeNavigationBar[`name == "TRASH"`]',
+  SENT_HEADER: '-ios class chain:**/XCUIElementTypeNavigationBar[`name == "SENT"`]',
   CREATE_EMAIL_BUTTON: '-ios class chain:**/XCUIElementTypeButton[`label == "+"`]',
-  INBOX_HEADER: '-ios class chain:**/XCUIElementTypeStaticText[`label == "INBOX"`]'
+  INBOX_HEADER: '-ios class chain:**/XCUIElementTypeStaticText[`label == "INBOX"`]',
```

Comment:
Let's use accessibility identifiers for these.

Wouldn't the tests all break if one email subject had "TRASH" or "+" in it?

Reply:
@tomholub I created new issue for this